mapie/classification.py

```diff
@@ -6,7 +6,8 @@
 import numpy as np
 from joblib import Parallel, delayed
 from sklearn.base import BaseEstimator, ClassifierMixin, clone
-from sklearn.model_selection import BaseCrossValidator, train_test_split
+from sklearn.model_selection import (BaseCrossValidator, train_test_split,
```

Comment:
Don't forget to use `isort` at the end to have the imports correctly ordered.

Reply:
Louis has suffered so much from this type of comments that he is now a psycho just like me <3

Reply:
So proud

Reply:
Done :)